node-manager/src/error.rs

```diff
@@ -200,6 +203,21 @@ impl From<MutinyError> for MutinyJsError {
     }
 }
 
+impl From<bdk::Error> for MutinyJsError {
+    fn from(e: bdk::Error) -> Self {
```

Comment:
I still don't think we need this because it should be turning into `MutinyError` instead of `MutinyJsError` directly.